config-examples/ph-config.json

```diff
@@ -2,15 +2,15 @@
   "BASE_URL": "coronastatus.ph",
   "COUNTRY_CODE": "ph",
   "DB_PATH": "./covid_db",
-  "LOCALE": "en-US",
+  "LOCALE": "en-PH",
```

Comment:
This will be overwritten by https://github.com/BustByte/coronastatus/pull/547, to we should do these changes there as well.